v4/export/ir_impl.go

```diff
@@ -193,17 +193,24 @@ func splitTableDataIntoChunks(
 		return
 	}
 
-	var max uint64
-	var min uint64
-	if max, err = strconv.ParseUint(smax.String, 10, 64); err != nil {
+	var max int64
+	var min int64
+	if max, err = strconv.Parseint(smax.String, 10, 64); err != nil {
```

Comment:
```suggestion
	if max, err = strconv.ParseInt(smax.String, 10, 64); err != nil {
```

and further more using `ParseInt` rather than `ParseUint` will make it fail with for unsigned PK column with value 18446744073709551615